setup.php

```diff
@@ -481,229 +477,223 @@ function cloneClicked() {
+                                </div>
+                            </div>
+                            <div class="col-sm-4">
+                                <div class="clearfix form-group">
+                                    <div class="label-div">
+                                        <label class="control-label font-weight-bold" for="dbname">Database Name:</label> <a href="#dbname_info"  class="info-anchor icon-tooltip"  data-toggle="collapse" ><i class="fa fa-question-circle" aria-hidden="true"></i></a>
+                                    </div>
+                                    <div>
+                                        <input name='dbname' id='dbname' type='text' class='form-control' value='openemr'>
+                                    </div>
                                 </div>
                                 <div id="dbname_info" class="collapse">
                                     <a href="#dbname_info" data-toggle="collapse" class="oe-pull-away"><i class="fa fa-times oe-help-x" aria-hidden="true"></i></a>
                                     <p>This will be the name of the OpenEMR database in MySQL.
-									<p>'openemr' is the recommended name.
-									<p>This database will contain patient data as well as data pertaining to the OpenEMR installation.
+                                    <p>'openemr' is the recommended name.
+                                    <p>This database will contain patient data as well as data pertaining to the OpenEMR installation.
                                 </div>
-								</div>
                             </div>
                         </div>
-                        <div class="row">
-                            <div class="col-12 ">
-                                <div class="col-sm-4">
-                                    <div class="clearfix form-group">
-                                        <div class="label-div">
-                                            <label class="control-label" for="login">Login Name:</label> <a href="#login_info"  class="info-anchor icon-tooltip"  data-toggle="collapse" ><i class="fa fa-question-circle" aria-hidden="true"></i></a>
-                                        </div>
-                                        <div>
-                                            <input name='login' ID='login' type='text' class='form-control' value='openemr'>
-
-                                        </div>
+                        <div class="ml-2 row">
+                            <div class="col-sm-4">
+                                <div class="clearfix form-group">
+                                    <div class="label-div">
+                                        <label class="control-label font-weight-bold" for="login">Login Name:</label> <a href="#login_info"  class="info-anchor icon-tooltip"  data-toggle="collapse" ><i class="fa fa-question-circle" aria-hidden="true"></i></a>
```

Comment:
fyi, control-label is no longer a class in BS4 from all I can tell. They now use a custom-control-* class which most likely doesn't apply to the use case here.

Reply:
Thanks for pointing that out. Did seem odd that it didn't do anything anymore (and lost the bold setting).

Reply:
They don't do it anymore cause of the old saying "the less classes you have to use the better" 😂

Reply:
just brought in a commit that removed this class from the setup.php script